src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -46,7 +46,7 @@ static const TypeMapping dnnlFCTypeMapping {
     {{_f32 | _bf16 | _f16, _any, _any, _i8 | _u8},       pt(bypass(), bypass(), use<0>(), use<0>())},
     // compresses float weights which do not match input data precision
     {{_f32, _half_float, _any, _any | _any},             pt(bypass(), bypass(), use<0>(), use<0>())},
-    {{_bf16, _f16, _any, _any | _any},                   pt(bypass(), bypass(), use<0>(), use<0>())},
+    {{_bf16, _f16 | _f32, _any, _any | _any},            pt(bypass(), bypass(), use<0>(), use<0>())},
```

Comment:
Shouldn't we change `dnnlConvolutionTypeMapping` as well? It looks like for some configurations "convolution_1x1_dnnl" may be selected during the initialization phase and alter the ports precisions as well.
NIT. Probably this change should be made in the first element `{{_bf16, _bf16, _any, _bf16 | _f32},` -> `{{_bf16, _bf16 | _f32, _any, _bf16 | _f32}` as the currently modified section is dedicated to "compressed weights".

Reply:
Agree. Updated